### PR TITLE
new caseless and 17mm firearm (making use of the unused)

### PR DESCRIPTION
**Pull request by Karlov357**
## About The Pull Request
<details>
<summary>
	TL:DR adds a new firearm using the underused 10x24 and 17mm cartridges, only obtainable via gun crafting (due to lack of how to code disks/cargo console). Parts are rare enough that it should take ~1-2 hours of work to obtain leaving other early game firearm options untouched and balance maintained for ballistics.
</summary>
<hr>
	
       This PR is for the addition of a new rifle, the 'mercurian anti-atmos rifle', which is based on the current solmarine designs but in the form of a specialists weapon for breaching atmosphere holding areas from an exterior vacuum. Stats wise the rifle is roughly on par with the specops SMG using 10x24 caseless with the addition of a single 17mm rolled shot underslung rifle with a SEVERE recoil impulse that keeps shots off target for ~2-3 seconds after firing from personal testing. The Mercurian should be quite good in terms of PVE, essentially acting as a slightly better AK but with a 40 round mag and no chance to up magazine size, which keeps the firearm balanced with most other options and in relation with the time it takes to unlock the ability to craft said firearm.
       
       A separate Mecurian.dm file was added to keep the currently bloated Solmarine.dm free of clutter, though the solmarine.dm file was tweaked to allow for the crafting of the mercurian (allowance of rubber grip and 10x24 barrel to be attached to the frame) and lastly the mercurian.dmi icon suite file was added.
<hr>
</details>

## Changelog
:cl:
add: Added new things
code: changed some code
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/automatic/mercurian.dm [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+/obj/item/gun/projectile/automatic/mercu/examine(mob/user)
+	..()
+	if(borerifle.loaded.len)
+		to_chat(user, "\The [borerifle] has \a [borerifle.chambered] loaded.")
+	else
+		to_chat(user, "\The [borerifle] is empty.")
+
+/obj/item/gun/projectile/automatic/underslung/borerifle
+	name = "Unnderslung large bore rifle"
```
Comment (benj8560) [inline code/modules/projectiles/guns/projectile/automatic/mercurian.dm:83]:
```suggestion
	name = "Underslung large bore rifle"
```
extra N there I don't think you ment.
